### PR TITLE
Small Ol7 updates, rule additions, and incorporation in JINJA statements

### PR DESCRIPTION
**Pull request by Xeicker**
#### Description:

- Update OL7 STIG Ids in `mount_option_dev_shm_nodev` & `mount_option_dev_shm_nosuid`
- Update `grub2_password` rule for OL7 to only check configuration in user.cfg, as done in ol8
- Update `kernel_module_disabled` template for OL products, so OVAL doesn't check `modprobe.conf`
- Replace the rule `auditd_data_retention_space_left` with `auditd_data_retention_space_left_percentage` in OL7 STIG profile
- Replace `configure_firewalld_rate_limiting` rule with `sysctl_net_ipv4_tcp_invalid_ratelimit` in OL7 STIG profile
- Add rule `package_firewalld_installed` to OL7 STIG profile

#### Rationale:

- All this with the intention to align better OL7 STIG profile with DISA's requirements

#### Review Hints:

- All this changes just take advantage of existing content. So I don't see much potential to break something

### CODE REVIEW

File: shared/templates/kernel_module_disabled/oval.template [hunk 1]
```diff
@@ -17,7 +17,7 @@
       <criterion test_ref="test_kernmod_{{{ KERNMODULE }}}_disabled"
       comment="kernel module {{{ KERNMODULE }}} disabled in modprobe.d" />
       {{% endif %}}
-{{% if "ubuntu" not in product and "rhel" not in product%}}
+{{% if "ubuntu" not in product and "rhel" not in product and "ol" not in product%}}
```
Comment (marcusburghardt) [inline shared/templates/kernel_module_disabled/oval.template:20]: Could you simplify using lists?

File: shared/templates/kernel_module_disabled/oval.template [hunk 2]
```diff
@@ -68,7 +68,7 @@
   </ind:textfilecontent54_object>
 {{% endif %}}
 
-{{% if "ubuntu" not in product and "rhel" not in product %}}
+{{% if "ubuntu" not in product and "rhel" not in product and "ol" not in product %}}
```
Comment (marcusburghardt) [inline shared/templates/kernel_module_disabled/oval.template:71]: Could you simplify using lists?

**Review by freddieRv — APPROVED** [review]
Apart from Marcus' comments, these changes LGTM.

File: shared/templates/kernel_module_disabled/oval.template (hunk 1 above)
Comment (marcusburghardt) [reply shared/templates/kernel_module_disabled/oval.template:20]: Understood, lets keep it simple. Thanks

**Review by marcusburghardt — APPROVED** [review]

**Pull request merged**